tests/json_parser_test.rb

```diff
@@ -84,6 +84,7 @@ def test_parse_simple_objects
     assert_equal({ "a" => 23 }, parse('  { "a"  : 23  } '))
     assert_equal({ "a" => 0.23 }, parse(' { "a"  :  0.23 }  '))
     assert_equal({ "a" => 0.23 }, parse('  {  "a"  :  0.23  }  '))
+    assert_equal({ "" => 123 }, parse('{"":123}'))
```

Comment:
Technically this test is a duplicate given https://github.com/flori/json/blob/master/tests/fixtures/pass1.json#L15 however it seems reasonable to have an explicit test for this case here too.